packages/rolldown/tests/fixtures/output/format/iife/_config.ts

```diff
@@ -12,6 +12,7 @@ export default defineTest({
     expect(output.output[0].code).toMatchInlineSnapshot(`
       "var myModule = (function() {
 
+
```

Comment:
Here could be better add a `globals` options to test binding workflow.

Reply:
I added tests for not only `globals` but also `exports` in iife.